src/Controller/DefaultController.php

```diff
@@ -250,87 +189,42 @@ public function generatePublicKeyAction(Request $request)
         return $this->redirect($this->generateUrl('manageParameters'));
     }
 
-    public function trans($msg, $params = array(), $domain = 'BinovoElkarBackup')
-    {
-        return $this->translator->trans($msg, $params, $domain);
-    }
-
     /**
      * @Route("/client/{id}/delete", name="deleteClient", methods={"POST"})
      */
-    public function deleteClientAction(Request $request, $id)
+    public function deleteClientAction(Request $request, ClientService $clientService, $id)
     {
-        $access = $this->checkPermissions($id);
-        if ($access == False) {
-            return $this->redirect($this->generateUrl('showClients'));
-        }
-        
         $t = $this->translator;
-        $db = $this->getDoctrine();
-        $manager = $db->getManager();
-        $repository = $db->getRepository('App:Client');
-        $manager = $db->getManager();
-        $client = $repository->find($id);
-        $queue = $db->getRepository('App:Queue')->findAll();
-        foreach ($queue as $item) {
-            if ($item->getJob()->getClient()->getId() == $id) {
-                $response = new JsonResponse(array(
-                    'error' => true,
-                    'msg' => $t->trans(
-                        'Could not delete client %clientName%, it has jobs enqueued.',
-                        array('%clientName%' => $client->getName()),
-                        'BinovoElkarBackup'
-                    ),
-                    'data' => array($id)
-                ));
-                $this->err(
-                    'Could not delete client %clientName%, it has jobs enqueued.',
-                    array('%clientName%' => $client->getName()),
-                    array('link' => $this->generateClientRoute($id))
-                );
-                $manager->flush();
-                return $response;
-            }
-        }
-        
         try {
-            $manager->remove($client);
-            $msg = new Message(
-                'DefaultController',
-                'TickCommand',
-                json_encode(array(
-                    'command' => "elkarbackup:delete_job_backups",
-                    'client' => (int) $id
-                ))
-            );
-            $manager->persist($msg);
-            $this->info(
-                'Client "%clientid%" deleted',
-                array('%clientid%' => $id),
-                array('link' => $this->generateClientRoute($id))
-            );
-            $manager->flush();
+            $clientService->delete($id);
             $response = new JsonResponse(array(
                 'error' => false,
                 'msg' => $t->trans(
-                    'Client %clientName% deleted successfully.',
-                    array('%clientName%' => $client->getName()),
+                    'Client %clientId% deleted successfully.',
+                    array('%clientId%' => $id),
```

Comment:
This message is for the user in WUI, I think we should keep using client name so that it is gentler for the user.